jobs/refresh-hiv-data.yml

```diff
@@ -1,15 +1,13 @@
 # For the HIV Server, we only execute the "hiv", "ovc", and "hivmigration" ETL jobs
-
 type: "job-pipeline"
 description: "Refresh HIV Data for ${siteName}"
 parameters:
   siteName: "hiv"
-  partitionNum: "8"
+  partitionNum: "1"
```

Comment:
I find this job very confusing.  Do we need this job?  Where is it running / used?  If this is for a specific server, then I think we need to establish better naming conventions / organization of job files to make this clear.  Because having a "refresh-hiv-tables.yml" and "refresh-hiv-data.yml" is confusing.  Maybe this particular job should be renamed to "refresh-haitihivtest-server.yml" or something, similar to "refresh-humci"?